src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -1750,39 +1784,62 @@ private void TestConflictResolverWins(bool returnRemoteDoc)
             }
         }
 
-        private void CreateReplicationConflict(string id)
+        private void CreateReplicationConflict(string id, bool checkFlags = false)
```

Comment:
This is not good for long term maintenance.  I guarantee everyone will forgot what "checkFlags" means here.  Maybe pass in a nullable set of expected flags (for mercury time sake it can be this way though).

Reply:
I will cherry pick this into master and replace this with your suggestion.